src/main/java/de/tum/in/www1/artemis/web/rest/ModelingSubmissionResource.java

```diff
@@ -323,9 +323,9 @@ else if (!authCheckService.isAtLeastInstructorForExercise(exercise)) {
      * @param participationId the participationId for which to find the submission and data for the modeling editor
      * @return the ResponseEntity with the submission as body
      */
-    @GetMapping("/modeling-editor/{participationId}")
+    @GetMapping("/participations/{participationId}/latest-submission")
```

Comment:
The naming seems very generic. Shouldn't `modeling` be in it somewhere since it only supports modeling exercises?

Reply:
True. Missed it somehow. I renamed it to `latest-modeling-submission`